ceph/rados/core_workflows.py

```diff
@@ -3399,3 +3399,66 @@ def log_cluster_health(self):
         health_detail, _ = self.node.shell(args=["ceph health detail"])
         log.info(f"\n****\n Cluster health detail: \n {health_detail} \n ****")
         return health_detail
+
+    def create_ecpool_inconsistent_obj(
+        self, pool_object, client_node, pool_name, no_of_objects
+    ):
+        """
+        The method converts the object into inconsistent object in EC pool
+        The logic implemented in the code is-
+        1. Get the target osd and pg_id
+        2. Get the object details by using the ceph-objectstore-tool
+        3. Remove the “hinfo_key” of the object
+        4. perform scrub
+        Args:
+            pool_object: pool object
+            client_node: client object
+            pool_name: pool name
+            no_of_objects: no of objcts to convert in to inconsistent objects
+        Returns: After converting the object in to inconsistent,method returns the pg id
+                 If it fail returns None
+        """
+        for obj_num in range(no_of_objects):
```

Comment:
Let's create all 4 objects, and then continue with the workflow.

Reply:
Code implemented.